pyface/ui/qt4/tasks/dock_pane.py

```diff
@@ -185,7 +185,8 @@ def _set_visible(self, event):
 
     def _receive_dock_area(self, area):
         with self._signal_context():
-            self.dock_area = INVERSE_AREA_MAP[int(area)]
+            if int(area) not in INVERSE_AREA_MAP:
+                self.dock_area = INVERSE_AREA_MAP[int(area)]
```

Comment:
I'm confused by this change. If `int(area) not in INVERSE_AREA_MAP` is True, then the key `int(area) doesn't exist in the dict `INVERSE_AREA_MAP` and ` `INVERSE_AREA_MAP[int(area)]` should raise an exception. Right?

Reply:
Yep, that's an error - apparently I can't copy code correctly :)

Not sure why it didn't crash horribly when I was testing.

```suggestion
            if int(area) in INVERSE_AREA_MAP:
                self.dock_area = INVERSE_AREA_MAP[int(area)]
```